src/components/LicenseAmendmentList/LicenseAmendmentList.js

```diff
@@ -26,40 +27,74 @@ export default class LicenseAmendmentList extends React.Component {
       id: PropTypes.string.isRequired,
     }),
     renderStatuses: PropTypes.bool,
+    renderWarnings: PropTypes.bool,
+    renderNotes: PropTypes.bool,
   }
 
+  renderStatusMismatchWarnings(amendment) {
+    const statusInAgreement = get(amendment, 'statusForThisAgreement.value');
+    return getConflictWarnings.amendmentWarning(amendment, statusInAgreement);
+  }
+
+
   render() {
     const {
       amendments,
       id,
       license,
       renderStatuses,
+      renderWarnings,
+      renderNotes,
     } = this.props;
 
+    let columns = ['warning', 'name', 'status', 'startDate', 'endDate', 'note'];
+    columns = renderStatuses ? columns : columns.filter(column => column !== 'status');
+    columns = renderWarnings ? columns : columns.filter(column => column !== 'warning');
+    columns = renderNotes ? columns : columns.filter(column => column !== 'note');
+
     return (
       <MultiColumnList
         columnMapping={{
-          note: '',
+          warning: '',
+          note: <FormattedMessage id="ui-agreements.note" />,
           name: <FormattedMessage id="ui-agreements.license.amendment" />,
           status: <FormattedMessage id="ui-agreements.status" />,
           startDate: <FormattedMessage id="ui-agreements.license.prop.startDate" />,
           endDate: <FormattedMessage id="ui-agreements.license.prop.endDate" />,
         }}
+        columnWidths={{
+          note: '350px'
+        }}
         contentData={amendments}
         formatter={{
-          note: a => (a.note ? <InfoPopover contentClass={css.note} content={a.note} /> : ''),
+          warning: a => (
+            this.renderStatusMismatchWarnings(a) ?
+              <Tooltip
+                id={`warning-tooltip-${a.id}`}
+                text={this.renderStatusMismatchWarnings(a)}
+                placement="left"
+              >
+                {({ ref, ariaIds }) => (
+                  // For the time being I'm using the workaround of a <span> while we wait to see what can/should be added to the 'Icon' component
+                  <Icon
+                    icon="exclamation-circle"
+                    iconClassName={css.tooltipIcon}
+                    ref={ref}
+                    aria-label={ariaIds.text}
```

Comment:
```suggestion
                    aria-labelledby={ariaIds.text}
```
`ariaIds` provides `id`s of the elements that will provide the label/description, not the label/description itself. So we need to pass the id to [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) (or `aria-describedby` in other cases), not `aria-label`.

Reply:
Ah ok, change made